src/platform/src/Bridge/Voyage/Voyage.php

```diff
@@ -19,17 +19,34 @@
  */
 class Voyage extends Model
 {
+    /** Supported dimensions: 2048, 1024, 512, or 256 */
```

Comment:
It's useful to have this information here. Could we validate the `dimensions` option in the constructor and indicate the dimensions supported in case of error?

Reply:
I'm usually a bit reluctant to bring in those validations because we don't really own that and don't rely on that ourselves, but it is only with Voyage.

Change scenario could be that they open up or change the support, the lib user want to leverage it, but is limited by our design - and we would need to patch and release to enable them.

We should check tho what happens if the user submits wrong options to the API, and if the error message gets back to the developer - that one of our weak spots still ...

Reply:
@natewiebe13 by any chance, did you check that already?

Reply:
@chr-hertel a `\Symfony\AI\Platform\Exception\RuntimeException` is thrown with `Response does not contain embedding data.`